samples/kvssink_gstreamer_sample.cpp

```diff
@@ -795,7 +819,8 @@ int gstreamer_init(int argc, char *argv[], CustomData *data) {
 
     // Create a GStreamer timer to generate and put fragment metadata tags every 2 seconds
```

Comment:
Is the purpose of this sample to show sending an image event every two seconds? Is the number of images specified at the level of the stream?

Reply:
The purpose is to show that this can be done repeatedly per fragment and not just once. Since this is just an example, we put the event until we reach the limit